coalib/results/SourcePosition.py

```diff
@@ -36,3 +37,6 @@ def __json__(self, use_relpath=False):
         if use_relpath:
             _dict['file'] = relpath(_dict['file'])
         return _dict
+
+    def __str__(self):
+        return self.filename
```

Comment:
this should show the filename, line number if it exists, and the column number if it also exists

separated by colons.

Reply:
I guess the CLI section is parsed by `CliParsing.py` and there are no changes made there, so no line number.